src/FindSegmentsMsg.chpl

```diff
@@ -132,11 +132,9 @@ module FindSegmentsMsg
               [(u, s, i) in zip(ukeylocs, permKey, paD)] if ((i > paD.low) && (permKey[i-1] != s))  { u = true; }
             }
             when "str" {
-              // TODO remvove legacy_placeholder
-              var (myNames1,legacy_placeholder) = name.splitMsgToTuple('+', 2);
               fsLogger.info(getModuleName(),getRoutineName(),getLineNumber(),
-                                  "findSegmentsMessage myNames1: {} legacy_placeholder: {}".format(myNames1,legacy_placeholder));
-              var str = getSegString(myNames1, st);
+                                  "findSegmentsMessage myNames1: {}".format(name));
```

Comment:
the logging message should be updated to read `"findSegmentsMessage name: {}".format(name)` to account for the change from `myNames1` to `name`

Reply:
Completely missed that. It has been updated.